src/test/ui/issues/issue-28344.stderr

```diff
@@ -1,8 +1,8 @@
-error[E0191]: the value of the associated type `Output` (from the trait `std::ops::BitXor`) must be specified
+error[E0191]: the value of the associated type `Output` (from trait `std::ops::BitXor`) must be specified
   --> $DIR/issue-28344.rs:4:17
    |
 LL |     let x: u8 = BitXor::bitor(0 as u8, 0 as u8);
-   |                 ^^^^^^^^^^^^^ associated type `Output` must be specified
+   |                 ^^^^^^ help: specify the associated type: `BitXor<Output = Type>`
```

Comment:
This is the case I'm protecting against when figuring out that there's a path with only one segment in the trait ref.